docs/guides/error.md

```diff
@@ -6,7 +6,7 @@ sidenav: doc-side-guides-nav.html
 title: Error Handling
 type: markdown
 ---
-<p class="lead"> This page describes how gRPC deals with errors, including gRPC's built-in error codes.</p>
+<p class="lead"> This page describes how gRPC deals with errors, including gRPC's built-in error codes. Example code in different languages can be found [here](https://github.com/avinassh/grpc-errors).</p>
```

Comment:
this is a html block so, the markdown markup to linkify does not work.
Could you please change this to using href to add the link?

Reply:
Fixed.